src/lib/reducers/diva-reducer.js

```diff
@@ -57,7 +57,6 @@ export default (state = initialState, action) => {
           ...state.sessions,
           [action.viewId]: {
             irmaSessionType: action.irmaSessionType,
-            isPolling: false,
```

Comment:
Is there no case at all where the `isPolling` flag is useful in UI's? Or can it be derived from other props?